HPXMLtoOpenStudio/resources/hpxml_defaults.rb

```diff
@@ -284,7 +284,7 @@ def self.apply_utility_bill_scenarios(runner, hpxml)
     hpxml.header.utility_bill_scenarios.each do |scenario|
       hpxml_doc = hpxml.to_oga if hpxml_doc.nil?
 
-      if HPXML::has_fuel(hpxml_doc, HPXML::FuelTypeElectricity)
+      if HPXML::has_fuel(hpxml_doc, HPXML::FuelTypeElectricity) && scenario.elec_tariff_filepath.nil?
```

Comment:
Can you update the tests in test_defaults.rb to demonstrate that when there's a tariff file provided, the defaults below are not applied?